<launch>

  <node name="motionplanning_api" pkg="MyDemo" type="motionplanning_api" respawn="false" output="screen">
    <rosparam command="load" file="$(find gluon_moveit_config)/config/kinematics.yaml"/>
    <param name="/planning_plugin" value="ompl_interface/OMPLPlanner"/>
    <rosparam command="load" file="$(find gluon_moveit_config)/config/ompl_planning.yaml"/>
  </node>

</launch>
